pkg/blockstorage/vmware/vmware.go

```diff
@@ -16,6 +17,11 @@ import (
 
 var _ blockstorage.Provider = (*fcdProvider)(nil)
 
+const (
+	noDescription   = ""
+	defaultWaitTime = 10 * time.Minute
```

Comment:
is there any guidance on how long this might take? 10 Minutes seems like a low timeout. I'd say let's increase this. Do you know if task.Wait honors context cancellation? if so we can effectively change the timeout in our tests vs prod.

Reply:
@tdmanv I don't have definite answers for your questions. 
I am not sure, `task.Wait` honors the context cancellation.
However, in my manual testing, these were fast operations, they were completed in seconds.

Reply:
10 minutes works for me for now